termbox/compat.go

```diff
@@ -25,17 +25,32 @@ import (
 var screen tcell.Screen
 var outMode OutputMode
 
+func GetScreen() tcell.Screen {
+	return screen
+}
+func SetScreen(s tcell.Screen) {
+	screen = s
+}
```

Comment:
Again, this is mixing and matching tcell and termbox.  Thats something I'm not entirely comfortable with.  At the minimum this needs to be isolated so that it won't pollute the termbox namespace.

Reply:
The original app is speaking in terms of the termbox namespace, but with a port the goal is to use tcell. So both namespaces are in use with the goal of eventually moving to tcell as things stabilize. There's no concern about pollution because I'm not changing the termbox-go library in any way. Sorry I don't follow.  Mixing and matching is inevitable with a port, but only benefits tcell because termbox applications can more easily start to use tcell.

Reply:
One reason I'm not worried about the mixing and matching because as an application author I have to be aware of which (termbox or tcell) I'm using anyway. The events provided by tcell don't match the events provided by termbox. The keycodes and Ch are different.

Reply:
So I'm worried here about the notion of an "embrace and extend" -- if we publish names into the termbox namespace, and termbox ever wants to use the same names with a different use or semantic, then I've just scewed over anyone using them.

The names that are not strictly termbox compatible need to be isolated into a separate namespace.

I'd suggest a package like "termbox/tcell_migration" or somesuch.

That makes it clear that these names are to facilitate migrating from termbox to tcell.

Having said that if these are just transition APIs, do they need to be in this package at all?  (I'll go back over the code review to see if that's the case.)

Reply:
Looking at MakeStyle, I'd just lift the implementation, and stick it in your transition library.  Its not a good long term solution, but you're transitioning anyway.

GetScreen and SetScreen are harder.

I think the right way to handle this is to have a function GetCurrentScreen() in tcell, and NewScreen() should initialize the backing variable for that.

This may actually help other use cases (reinitialization.)